be2-scala/src/test/scala/ch/epfl/pop/model/objects/ChannelSuite.scala

```diff
@@ -61,15 +61,15 @@ class ChannelSuite extends FunSuite with Matchers {
   test("LaoId extraction channel test") {
     val laoId = "base64_lao_id";
     def channel = Channel(Channel.ROOT_CHANNEL_PREFIX + Base64Data.encode(laoId))
-    val expected = laoId.getBytes()
+    val expected = Base64Data.encode(laoId)
```

Comment:
what's the difference between this test and `Real LaoId extraction fails with wrong channel structure (not base64Data)` ?

Reply:
Here we do not forget to encode it